app/src/main/java/nl/tudelft/bbw/blockchain/User.java

```diff
@@ -30,6 +30,18 @@ public User(String name, String iban, EdDSAPublicKey publicKey) {
     }
 
     /**
+     * Constructor for user class
+     * User class used by crawler, iban in this user class is equal to the string representation of the read public key
+     *
+     * @param name given name
+     * @param iban given iban
+     */
+    public User(String name, String iban) {
+        this.name = name;
+        this.iban = iban;
+    }
+
+    /**
```

Comment:
There is no other solution, since the public key cannot be stored in another way.